modules/appeals_api/spec/fixtures/v2/invalid_200995_evidence.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
I don't think adding these fixture files is the way I'd like to test these scenarios. Notice that most of the controller tests take valid fixture file data (e.g. `extra_data`) and manipulate it just before sending the requests as part of the `it`s themselves. This means fewer fixture files to update when schemas change, and manipulation for specific purposes is easier to change in-spec rather than going to an external file. 

To dig into the philosophy a bit here -
The existing fixtures are "worthy" of being fixtures because they set up the baseline for the multitude of other tests that can use them - they're not  necessarily supposed to exist for one specific test. I think the `characters` fixtures are the only place we've broken that philosophy due to character encoding issues between `rb` files and `txt` files.

All that said, I won't block the merge for this since it'll keep va.gov moving forward and it's easy enough to refactor as another fast-follow PR :)